app/Http/Controllers/User/AuthController.php

```diff
@@ -49,10 +65,15 @@ public function registerUser(RegisterUserRequest $request): RedirectResponse
     public function verified(User $register, string $token): RedirectResponse
     {
         try {
+            $timenow = Carbon::now()->format('Y-m-d H:i:s');
+            $diff = Carbon::parse($timenow)->diffInMinutes(Carbon::parse($register->mail_verify_at));
```

Comment:
1. Đã convert now to string xong lại convert ngược lại carbon?
2. mail_verify_at được set lúc nào?